azerothcore/azerothcore-wotlk
# fix(DB/Creature): removed skull visual modification

Removing skull visual modification from previous Dungeons Boss NPC's.

To test:

1. Apply SQL
2. Restart server & delete your cache
3. Go to any dungeon (Classic, TBC or WoTLK) and check if boss have skull or level number.
